### DISCUSSION TITLE
the latest version of fastapi install pydantic V1, i expected V2 ?

### DISCUSSION THREAD

**Discussion by badnanna** [fastapi/fastapi#11594] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/pydantic/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
row = models.Ensign(**data.model_dump())
```

### Description

model_dump() work only in pydantic V2, if you install latest version of Fastapi you can't do that line.

### Operating System

Linux

### FastAPI Version

lastest

### Pydantic Version

v1

### Python Version

3.12

**Comment by YuriiMotov** [comment, marked as answer]
Check whether you have a clean environment before you install FastAPI. You probably had Pydantic v1 installed before you ran FastAPI installation.

Just checked that pydantic-2.7.1 is installed if you install FastAPI on clean environment
